packages/chart/src/Column.ts

```diff
@@ -74,12 +71,13 @@ export class Column extends XYAxis {
         let offset = 0;
         switch (host.xAxisType()) {
             case "ordinal":
+                host.paddingOuter(this.paddingOuter());
                 dataLen = host.bandwidth();
                 offset = -dataLen / 2;
                 break;
             case "linear":
             case "time":
-                dataLen = Math.max(Math.abs(host.dataPos(2) - host.dataPos(1)) * (100 - this._linearGap) / 100, dataLen);
+                dataLen = Math.max(Math.abs(host.dataPos(2) - host.dataPos(1)) * this.paddingOuter(), dataLen);
```

Comment:
@GordonSmith renaming `scalePadding` to `ordinalPaddingOuter` would affect these liner/time paddings... should I create another publish parameter for these? or...